analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -60,6 +60,10 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="RedundantArgument.CSharp12.cs" />
```

Comment:
Educational: why are we removing this specific file?

Reply:
I don't know: it was automatically added by Visual Studio for some reason.
I have removed it and all seems to compile and run correctly (because that file is actually just test data, and it's not included in the compilation of `SonarAnalyzer.UnitTest`, so removing it doesn't produce any visible effect).